test-app/src/components/debugger.tsx

```diff
@@ -307,7 +308,9 @@ export const Debugger = () => {
       {txId && (
         <Text textStyle="body.large" display="block" my={'base'}>
           <Text color="green" fontSize={1}>
-            Successfully broadcasted &quot;{txType}&quot;
+            <span data-testId={WalletPageSelectors.statusMessage}>
```

Comment:
Convention for enums is pascal case, requires update in multiple other cases
```suggestion
            <span data-testId={WalletPageSelectors.StatusMessage}>
```